app/controllers/games_controller.rb

```diff
@@ -19,7 +19,17 @@ def piece_update
 		@game = Game.find(params[:game_id])
 		@piece = Piece.find(params[:id])
 		@piece.update_attributes(:x_axis => params[:x_axis], :y_axis => params[:y_axis])
-		redirect_to game_path(@game)
+
+		if King.checked?(King.current_position)
+			render html: "<strong>King Checked</strong>".html_safe
```

Comment:
We might want to make separate pages rather than just inline html, but this gets the point across for now.